services/wallet/responses/router_suggested_routes.go

```diff
@@ -6,10 +6,9 @@ import (
 )
 
 type RouterSuggestedRoutes struct {
-	Uuid                  string                `json:"Uuid"`
-	Best                  routes.Route          `json:"Best,omitempty"`
-	Candidates            routes.Route          `json:"Candidates,omitempty"`
-	TokenPrice            *float64              `json:"TokenPrice,omitempty"`
-	NativeChainTokenPrice *float64              `json:"NativeChainTokenPrice,omitempty"`
-	ErrorResponse         *errors.ErrorResponse `json:"ErrorResponse,omitempty"`
+	Uuid          string                `json:"Uuid"`
+	Best          routes.Route          `json:"Best,omitempty"`
+	Candidates    routes.Route          `json:"Candidates,omitempty"`
+	UpdatedPrices *map[string]float64   `json:"UpdatedPrices,omitempty"`
```

Comment:
no need to make this a pointer, the default nil map will get discarded

https://pkg.go.dev/encoding/json
```
The "omitempty" option specifies that the field should be omitted from the encoding if the field has an empty value, defined as false, 0, a nil pointer, a nil interface value, and any empty array, slice, map, or string.

```

Reply:
good point! done